docs/cli/cmd-options.mdx

```diff
@@ -677,10 +711,18 @@ Format time as: relative, iso, raw.
 
 Time zone (IANA name).
 
+## tls-ca-data
+
+Data for server CA certificate. Can't be used with --tls-ca-path.
```

Comment:
The reason is that you're providing two ways to load the CA cert info, however this is in-line with Temporal CLI wording. No change recommended.